node/coinstacks/solana/api/src/controller.ts

```diff
@@ -73,9 +79,67 @@ export class Solana implements BaseAPI {
   @Response<ValidationError>(422, 'Validation Error')
   @Response<InternalServerError>(500, 'Internal Server Error')
   @Get('account/{pubkey}/txs')
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  async getTxHistory(@Path() pubkey: string, @Query() _cursor?: string, @Query() _pageSize = 10): Promise<TxHistory> {
-    return { pubkey } as TxHistory
+  async getTxHistory(
+    @Path() pubkey: string,
+    @Query() lastSignature?: string,
+    @Query() pageSize = 10
+  ): Promise<TxHistory> {
+    const urlParams = new URLSearchParams()
+    urlParams.append('api-key', RPC_API_KEY ?? '')
+    urlParams.append('limit', pageSize.toString())
+    if (lastSignature) {
+      urlParams.append('before', lastSignature)
+    }
+
+    try {
+      const { data } = await axiosNoRetry.get<RawTx[]>(
```

Comment:
why aren't we using the sdk?

Reply:
Because this endpoint is not in the SDK (https://github.com/helius-labs/helius-sdk/issues/28)